rpcs3/Emu/Cell/lv2/sys_rsx.cpp

```diff
@@ -230,6 +230,13 @@ s32 sys_rsx_context_attribute(s32 context_id, u32 package_id, u64 a3, u64 a4, u6
 
 	auto m_sysrsx = fxm::get<SysRsxConfig>();
 
+	if (!m_sysrsx)
+	{
+		// Hamilton's Great Adventure [NPUB30458] calls this function before calling allocate. Not sure if race condition.
+		sys_rsx.error("sys_rsx_context_attribute called before sys_rsx_context_allocate: context_id=0x%x, package_id=0x%x, a3=0x%llx, a4=0x%llx, a5=0x%llx, a6=0x%llx)", context_id, package_id, a3, a4, a5, a6);
+		return CELL_ESRCH;
```

Comment:
Just change to EINVAL to match real hw

Reply:
Fixed and rebased. Sorry I don't pay attention to github notifications.

Reply:
If handling that matches real console behavior then it shouldn't have the "Not sure if it's race condition" comment as it's correct